### PR TITLE
Removed unused imports, variables, functions

### PR DESCRIPTION
**Pull request by WillAyd**
flake8 identifies all of these as unused - figured an easy cleanup

### CODE REVIEW

File: tabcmd/commands/commands.py [hunk 1]
```diff
@@ -13,13 +13,6 @@ def __init__(self, args):
         self.personal_token = args.token
         self.logging_level = args.logging_level
 
-    @staticmethod
```
Comment (WillAyd) [inline tabcmd/commands/commands.py:16]: This is redefined later in the same module

File: tabcmd/commands/commands.py [hunk 2]
```diff
@@ -13,13 +13,6 @@ def __init__(self, args):
...
-    def exit_with_error(logger, error_message, exception=None):
```
Comment (jacalata) [inline tabcmd/commands/commands.py:17]: was this all done by an automated tool? This method is used in a couple of places, for instance in commands/auth/session.py.

Comment (WillAyd) [reply tabcmd/commands/commands.py:17]: Yea this is all from flake8

Comment (jacalata) [reply tabcmd/commands/commands.py:17]: ah, I missed that the method was defined twice, thought it was being deleted altogether.

**Review by jacalata — APPROVED** [review]

**Pull request merged**